bucket/pine64_updater.json

```
@@ -0,0 +1,19 @@
+{
+    "version": "1.0.3",
+    "license": "MIT",
+    "homepage": "https://github.com/pine64/pine64_updater",
+    "url": "https://github.com/pine64/pine64_updater/releases/download/1.0.3/pine64_updater_win64_1.0.3.zip",
+    "hash": "47bbced528e9219b70c22fa0d32a80c7fdb4a443edf06d8cc1396f04cc132a93",
+    "shortcuts": [
+        [
+            "pine64_updater.exe",
+            "Pine64 Updater"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/pine64/pine64_updater"
+    },
```

```suggestion
    "checkver": "github"
```

Since the homepage is a Github repo, we can just use `"checkver": "github"`.